InstituteBudget.php

```diff
@@ -1773,35 +1818,33 @@ public function redcap_every_page_top($project_id) {
     }
 	
 	public function redcap_survey_page($project_id, $record, $instrument, $event_id, $group_id, $survey_hash, $response_id, $repeat_instance) {
+        $this->initVars();
 		// replace schedule of event field in survey page with generated table
-		if ($instrument == $this->budget_table_instrument) {
+		if ($instrument == $this->getBudgetForm()) {
 			$this->replaceScheduleFields($record);
 		}
 		
 		// replace Go/No-Go field in survey page with generated table
-		if ($instrument == $this->gonogo_table_instrument) {
-			$this->replaceGoNoGoFields($record, $repeat_instance);
-		} 
+		//if ($instrument == $this->gonogo_table_instrument) {
+		//	$this->replaceGoNoGoFields($record, $repeat_instance);
+		//}
```

Comment:
Consider adding // TODOs: for these broken features that need

Reply:
Good idea, added.